backend/trafficrules/src/main/java/com/coffeejug/trafficrules/service/UserService.java

```diff
@@ -40,4 +64,23 @@ private UUID parseUUID(String uuid) {
             throw new BadRequestException("Invalid UUID " + uuid);
         }
     }
+
+    public long count() {

```

Comment:
What is the point of this method? Calling `userRepository.count()` is more readable, IMO